From 4a1ded0778b3799efe655deec50ac223dba0bf19 Mon Sep 17 00:00:00 2001
From: Nizan Zorea <nzorea@marvell.com>
Date: Wed, 23 Mar 2016 17:34:23 +0200
Subject: [PATCH 0979/1240] dts: a7040-rz: Changed 70x0-dev-info dtsi name to
 be specifically for 7040-rz

- this is done to separate 7040-rz and 70x0 dev info.
- a new dtsi for 70x0 will be created in following patches.

Change-Id: Ic647568fd57a74f7a55066f8cfb7ce861ca56e9e
Signed-off-by: Nizan Zorea <nzorea@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28481
Reviewed-by: Haim Boot <hayim@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_all_pci.dts    |   2 +-
 arch/arm/dts/armada-7040-rz-db_dop.dts        |   2 +-
 arch/arm/dts/armada-7040-rz-db_kr.dts         |   2 +-
 arch/arm/dts/armada-7040-rz-db_nand.dts       |   2 +-
 arch/arm/dts/armada-7040-rz-db_rgmii_only.dts |   2 +-
 arch/arm/dts/armada-7040-rz-db_router.dts     |   2 +-
 arch/arm/dts/armada-7040-rz-db_rxaui.dts      |   2 +-
 arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts  |   2 +-
 arch/arm/dts/armada-7040-rz-dev-info.dtsi     | 179 ++++++++++++++++++++++++++
 arch/arm/dts/armada-70x0-dev-info.dtsi        | 179 --------------------------
 10 files changed, 187 insertions(+), 187 deletions(-)
 create mode 100644 arch/arm/dts/armada-7040-rz-dev-info.dtsi
 delete mode 100644 arch/arm/dts/armada-70x0-dev-info.dtsi

diff --git a/arch/arm/dts/armada-7040-rz-db_all_pci.dts b/arch/arm/dts/armada-7040-rz-db_all_pci.dts
index 39ba23e..ae3c45e 100644
--- a/arch/arm/dts/armada-7040-rz-db_all_pci.dts
+++ b/arch/arm/dts/armada-7040-rz-db_all_pci.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (ALL_PCI)";
diff --git a/arch/arm/dts/armada-7040-rz-db_dop.dts b/arch/arm/dts/armada-7040-rz-db_dop.dts
index 4115c50..149663b 100644
--- a/arch/arm/dts/armada-7040-rz-db_dop.dts
+++ b/arch/arm/dts/armada-7040-rz-db_dop.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (DOP)";
diff --git a/arch/arm/dts/armada-7040-rz-db_kr.dts b/arch/arm/dts/armada-7040-rz-db_kr.dts
index 2f449fb..ed786ac 100644
--- a/arch/arm/dts/armada-7040-rz-db_kr.dts
+++ b/arch/arm/dts/armada-7040-rz-db_kr.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (KR)";
diff --git a/arch/arm/dts/armada-7040-rz-db_nand.dts b/arch/arm/dts/armada-7040-rz-db_nand.dts
index 24f0b49..27979f2 100644
--- a/arch/arm/dts/armada-7040-rz-db_nand.dts
+++ b/arch/arm/dts/armada-7040-rz-db_nand.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (NAND)";
diff --git a/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts b/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
index 99c6966..a8bfa4c 100644
--- a/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rgmii_only.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (RGMII_ONLY)";
diff --git a/arch/arm/dts/armada-7040-rz-db_router.dts b/arch/arm/dts/armada-7040-rz-db_router.dts
index eb0e0ce..3f53ff9 100644
--- a/arch/arm/dts/armada-7040-rz-db_router.dts
+++ b/arch/arm/dts/armada-7040-rz-db_router.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (ROUTER)";
diff --git a/arch/arm/dts/armada-7040-rz-db_rxaui.dts b/arch/arm/dts/armada-7040-rz-db_rxaui.dts
index 2a211b1..524f865 100644
--- a/arch/arm/dts/armada-7040-rz-db_rxaui.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rxaui.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (RXAUI)";
diff --git a/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts b/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
index 63627b6..6d74e01 100644
--- a/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
+++ b/arch/arm/dts/armada-7040-rz-db_rxaui_pci.dts
@@ -19,7 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0-rz.dtsi" /* include SoC device tree */
-#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
+#include "armada-7040-rz-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (RXAUI_PCI)";
diff --git a/arch/arm/dts/armada-7040-rz-dev-info.dtsi b/arch/arm/dts/armada-7040-rz-dev-info.dtsi
new file mode 100644
index 0000000..54c42a8
--- /dev/null
+++ b/arch/arm/dts/armada-7040-rz-dev-info.dtsi
@@ -0,0 +1,179 @@
+/*
+ * ***************************************************************************
+ * Copyright (C) 2015 Marvell International Ltd.
+ * ***************************************************************************
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation, either version 2 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * ***************************************************************************
+ */
+
+#include <dt-bindings/ddr/ddr_data.h>
+
+/ {
+	/* This should go only into devel boards */
+	compatible = "marvell,cp110";
+
+	sar {
+		compatible = "marvell,sample-at-reset";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x4c 0x4d 0x4e>;
+		chip_count = <3>;
+		bit_width = <5>;
+
+		freq {
+			key = "freq";
+			description = "CPU/DDR frequencies";
+			start-bit = <0>;
+			bit-length = <5>;
+			option-cnt = <4>;
+			options = "0xf", "2000/650 Mhz",
+				"0x11", "2000/650 Mhz",
+				"0x16", "1600/650 Mhz",
+				"0x17", "1200/650 Mhz";
+			default = <0x16>;
+			swap-bit;
+			status = "okay";
+		};
+		bootsrc {
+			key = "bootsrc";
+			description = "Boot source options";
+			start-bit = <5>;
+			bit-length = <2>;
+			option-cnt = <3>;
+			options = "0x0", "BootRom + SPI",
+				"0x1", "Companion Device",
+				"0x2", "SPI";
+			default = <0x1>;
+			status = "okay";
+		};
+		c2c_ihb {
+			key = "c2c_ihb";
+			description = "C2C IHB select";
+			start-bit = <7>;
+			bit-length = <1>;
+			option-cnt = <2>;
+			options = "0x0", "IHB",
+				"0x1", "C2C";
+			default = <0x1>;
+			status = "okay";
+		};
+		mpp_8 {
+			key = "MPP8";
+			description = "MPP8 - must be 1";
+			start-bit = <8>;
+			bit-length = <1>;
+			option-cnt = <1>;
+			options = "0x1", "MPP8";
+			default = <0x1>;
+			status = "okay";
+		};
+		boot_mode {
+			key = "boot_mode";
+			description = "Boot mode options";
+			start-bit = <9>;
+			bit-length = <6>;
+			option-cnt = <2>;
+			options = "0x32", "CP_SPI_1 IHB 24bits",
+				"0xf", "NAND IHB 8bits";
+			default = <0x32>;
+			status = "okay";
+		};
+	};
+
+	mpp-buses {
+		cp-mpp-bus0 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "spi1";
+			bank-name = "cp110";
+			pin-count = <4>;
+			pins = <13 3  14 3  15 3  16 3>;
+		};
+		cp-mpp-bus1 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "i2c";
+			bank-name = "cp110";
+			pin-count = <2>;
+			pins = <37 2  38 2>;
+		};
+		cp-mpp-bus2 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "uart0";
+			bank-name = "cp110";
+			pin-count = <2>;
+			pins = <29 0xA  30 0xA>;
+		};
+		cp-mpp-bus3 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "uart1";
+			bank-name = "cp110";
+			pin-count = <2>;
+			pins = <40 7  41 7>;
+		};
+		cp-mpp-bus4 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "sd";
+			bank-name = "cp110";
+			pin-count = <7>;
+			pins = <56 0xE  57 0xE  58 0xE 59 0xE  60 0xE  61 0xE  62 0xE>;
+		};
+		cp-mpp-bus5 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "tdm";
+			bank-name = "cp110";
+			pin-count = <12>;
+			pins = <0 4  1 4  2 4  3 4  4 4  5 4  6 4  7 4  8 4  9 4  10 4  11 4>;
+		};
+		cp-mpp-bus6 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "rgmii0";
+			bank-name = "cp110";
+			pin-count = <12>;
+			pins = <0 3  1 3  2 3  3 3  4 3  5 3  6 3  7 3  8 3  9 3  10 3  11 3>;
+		};
+		cp-mpp-bus7 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "rgmii1";
+			bank-name = "cp110";
+			pin-count = <11>;
+			pins = <45 2  46 1  47 1  48 1  49 1  50 1  51 1  52 1  53 1  54 1  55 1>;
+		};
+		cp-mpp-bus8 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "nand";
+			bank-name = "cp110";
+			pin-count = <14>;
+			pins = <13 2 15 1 16 1 17 1 18 1 19 1 20 1 21 1 22 1 23 1 24 1 25 1 26 1 27 1>;
+		};
+		cp-mpp-bus9 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "sata0";
+			bank-name = "cp110";
+			pin-count = <1>;
+			pins = <13 6>;
+		};
+		cp-mpp-bus10 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "sata1";
+			bank-name = "cp110";
+			pin-count = <1>;
+			pins = <28 9>;
+		};
+		cp-mpp-bus11 {
+			compatible = "marvell,mvebu-mpp-bus";
+			bus-name = "smi";
+			bank-name = "cp110";
+			pin-count = <2>;
+			pins = <32 7  34 7>;
+		};
+	};
+};
diff --git a/arch/arm/dts/armada-70x0-dev-info.dtsi b/arch/arm/dts/armada-70x0-dev-info.dtsi
deleted file mode 100644
index 54c42a8..0000000
--- a/arch/arm/dts/armada-70x0-dev-info.dtsi
+++ /dev/null
@@ -1,179 +0,0 @@
-/*
- * ***************************************************************************
- * Copyright (C) 2015 Marvell International Ltd.
- * ***************************************************************************
- * This program is free software: you can redistribute it and/or modify it
- * under the terms of the GNU General Public License as published by the Free
- * Software Foundation, either version 2 of the License, or any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- * ***************************************************************************
- */
-
-#include <dt-bindings/ddr/ddr_data.h>
-
-/ {
-	/* This should go only into devel boards */
-	compatible = "marvell,cp110";
-
-	sar {
-		compatible = "marvell,sample-at-reset";
-		#address-cells = <1>;
-		#size-cells = <0>;
-		reg = <0x4c 0x4d 0x4e>;
-		chip_count = <3>;
-		bit_width = <5>;
-
-		freq {
-			key = "freq";
-			description = "CPU/DDR frequencies";
-			start-bit = <0>;
-			bit-length = <5>;
-			option-cnt = <4>;
-			options = "0xf", "2000/650 Mhz",
-				"0x11", "2000/650 Mhz",
-				"0x16", "1600/650 Mhz",
-				"0x17", "1200/650 Mhz";
-			default = <0x16>;
-			swap-bit;
-			status = "okay";
-		};
-		bootsrc {
-			key = "bootsrc";
-			description = "Boot source options";
-			start-bit = <5>;
-			bit-length = <2>;
-			option-cnt = <3>;
-			options = "0x0", "BootRom + SPI",
-				"0x1", "Companion Device",
-				"0x2", "SPI";
-			default = <0x1>;
-			status = "okay";
-		};
-		c2c_ihb {
-			key = "c2c_ihb";
-			description = "C2C IHB select";
-			start-bit = <7>;
-			bit-length = <1>;
-			option-cnt = <2>;
-			options = "0x0", "IHB",
-				"0x1", "C2C";
-			default = <0x1>;
-			status = "okay";
-		};
-		mpp_8 {
-			key = "MPP8";
-			description = "MPP8 - must be 1";
-			start-bit = <8>;
-			bit-length = <1>;
-			option-cnt = <1>;
-			options = "0x1", "MPP8";
-			default = <0x1>;
-			status = "okay";
-		};
-		boot_mode {
-			key = "boot_mode";
-			description = "Boot mode options";
-			start-bit = <9>;
-			bit-length = <6>;
-			option-cnt = <2>;
-			options = "0x32", "CP_SPI_1 IHB 24bits",
-				"0xf", "NAND IHB 8bits";
-			default = <0x32>;
-			status = "okay";
-		};
-	};
-
-	mpp-buses {
-		cp-mpp-bus0 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "spi1";
-			bank-name = "cp110";
-			pin-count = <4>;
-			pins = <13 3  14 3  15 3  16 3>;
-		};
-		cp-mpp-bus1 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "i2c";
-			bank-name = "cp110";
-			pin-count = <2>;
-			pins = <37 2  38 2>;
-		};
-		cp-mpp-bus2 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "uart0";
-			bank-name = "cp110";
-			pin-count = <2>;
-			pins = <29 0xA  30 0xA>;
-		};
-		cp-mpp-bus3 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "uart1";
-			bank-name = "cp110";
-			pin-count = <2>;
-			pins = <40 7  41 7>;
-		};
-		cp-mpp-bus4 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "sd";
-			bank-name = "cp110";
-			pin-count = <7>;
-			pins = <56 0xE  57 0xE  58 0xE 59 0xE  60 0xE  61 0xE  62 0xE>;
-		};
-		cp-mpp-bus5 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "tdm";
-			bank-name = "cp110";
-			pin-count = <12>;
-			pins = <0 4  1 4  2 4  3 4  4 4  5 4  6 4  7 4  8 4  9 4  10 4  11 4>;
-		};
-		cp-mpp-bus6 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "rgmii0";
-			bank-name = "cp110";
-			pin-count = <12>;
-			pins = <0 3  1 3  2 3  3 3  4 3  5 3  6 3  7 3  8 3  9 3  10 3  11 3>;
-		};
-		cp-mpp-bus7 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "rgmii1";
-			bank-name = "cp110";
-			pin-count = <11>;
-			pins = <45 2  46 1  47 1  48 1  49 1  50 1  51 1  52 1  53 1  54 1  55 1>;
-		};
-		cp-mpp-bus8 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "nand";
-			bank-name = "cp110";
-			pin-count = <14>;
-			pins = <13 2 15 1 16 1 17 1 18 1 19 1 20 1 21 1 22 1 23 1 24 1 25 1 26 1 27 1>;
-		};
-		cp-mpp-bus9 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "sata0";
-			bank-name = "cp110";
-			pin-count = <1>;
-			pins = <13 6>;
-		};
-		cp-mpp-bus10 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "sata1";
-			bank-name = "cp110";
-			pin-count = <1>;
-			pins = <28 9>;
-		};
-		cp-mpp-bus11 {
-			compatible = "marvell,mvebu-mpp-bus";
-			bus-name = "smi";
-			bank-name = "cp110";
-			pin-count = <2>;
-			pins = <32 7  34 7>;
-		};
-	};
-};
-- 
1.9.1

